pkg/asset/installconfig/nutanix/nutanix.go

```diff
@@ -0,0 +1,276 @@
+// Package nutanix collects Nutanix-specific configuration.
+package nutanix
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	nutanixClientV3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
```

Comment:
Use all lowercase for the package name.
```suggestion
	nutanixclientv3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
```